Practice2/src/common-component.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/main.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div>
+        <h4>Buttons</h4>
+        <button type="button" class="btn btn-primary">Purchase</button>
+        <button type="button" class="btn btn-secondary">Purchase</button>
+    </div>
+    <div>
+        <h4>Cards</h4>
+        <div class="card">
+            <div class="card-body">
+                <h5 class="card-title card-title-primary">SILVER</h5>
+                <h4 class="card-text-number card-text-number-primary">$49</h4>
+                <h5 class="card-title">YEAR</h5>
```

Comment:
Should write the text `YEAR` or `SILVER` with the normal format `Year` or `Silver` and use CSS to style it.

Reply:
Fixed

Reply:
Are you sure this is fixed? I pulled the latest code and it's still the same.

Reply:
Yes, plz check my new commit